Sources/ProjectDescription/Tasks/Task.swift

```diff
@@ -0,0 +1,103 @@
+import Foundation
+
+public struct Task: Codable {
```

Comment:
I'd create a new framework other than `ProjectDescription` to contain all the automation-related models. I propose naming it `ProjectAutomation` to be consistent with `ProjectDescription` and capture what the framework contains. For now it's just a model, but we'll want to include abstractions for shelling-out and doing IO operations.